lessons/Projects/project-snack-habits/langs/en-us/index.adoc

```diff
@@ -0,0 +1,194 @@
+}
+
+We have set the form up with data verification for most questions to minimize the amount of cleaning you will have to do, but we would expect that the one word answers to "Why are you eating this snack?" will require some attention to address: 
+
+- typos
+- inconsistent capitalization
+- words that mean the same thing as each other and should be combined 
+- perhaps there are a number of nonsense words that might make more sense to be replaced by "idk" why I'm snacking?
+
+If you have the time, we encourage you to project the spreadsheet and tackle cleaning the data with your students, asking:
+
+- What inconsistencies do you see in the data?
+- How should we address them?
+
+Ideally, by the time you're done you could make a pie-chart of the "why" column and the breakdown of reasons would be informative. 
+}
+
+@lesson-instruction{
+- Now that we have finished collecting our data, let's see what we can learn about our habits!
+- Open the starter file I shared with you and use it to complete @handout{data-cycle-categorical.adoc}
+}
+
+@teacher{@right{@image{images/pie-chart-salty-sweet.png, 500}} For reference, here's an example of a display from one Bootstrap teacher's class. Consider the kinds of discussions your class might have if the data from your class came back looking like this.}
+
+=== Synthesize
+
+- What did you learn?
+- What new questions do you have?
+
+== Research: U.S. Snacking Habits Data
+
+=== Overview
+
+Students will gather information from studies about US American snacking habits and compare them to the data we've just gathered as a class.
+
+=== Launch
+
+@lesson-instruction{
+Do you think the sample we've made of our snack habits is representative of snacking in the United States?
+}
+
+We don't have to guess! There's plenty of research out there for us to look at. In fact, enough data has been collected about childhood obesity in the United States that soda machines and unhealthy snacks have been pulled from many schools.
+
+=== Investigate
+
+@lesson-instruction{
+- Open your favorite search engine (Google, Brave, DuckDuckGo, etc.) and type +
+"What do Americans eat for snack?”
+- Look for graphs and statistics that relate to the kinds of data we're collecting about our own snacking habits.
+}
+
+=== Synthesize
+
+Compare and contrast your findings from our class data and the research you've been doing about U.S. Data. 
+
+@scrub{Make a similar and different handout}
+
+== Analysis: A Statistical Question of Your Own
+
+=== Overview
+
+Building on their explorations of the class data and initial research about U.S. Snacking habits, students develop a statistical question to present their findings on.
+
+=== Launch
+
+Now that you've had some time to explore both our class data and the research that's been done about snacking habits in the U.S., it's time to identify a statistical question that is of particular interest to you and present your findings.
+
```

Comment:
Before students develop a statistical question, shouldn't they complete slides 2-6 (which don't require the question, I'm pretty sure?).  I think those slides will help them come up with a solid question, anyhow. I'd include instructions to do that. In fact, I would make the launch all about doing those slides, and then have them come up with a stat Q in the Investigate section.

Also, are they supposed to type their stat q on slide 1? If so, it should be explicitly stated. That said, I think it would make *more* sense to put the research qusetion on a slide added in between slide 6 and slide 7.

Also, it would be nice to give more guidance on developing statistical questions and offer a few examples. I imagine teachers would be grateful to have a few good ones to choose from, just in case.